docker/driver/base/Dockerfile

```diff
@@ -4,8 +4,8 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends curl ssh git libc-dev gcc openssl pkg-config libssl-dev \
     && rm -rf /var/lib/apt/lists/*
 
-#Install Rust
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+#Install Rust with version 1.36
```

Comment:
How is this enforced? Doesn't it always install the newest rust version?

Reply:
Yes it does. I tried to trigger a rebuild and only later realized that I have to push the docker image manully.
I will revert it